src/entities/Deployment.ts

```diff
@@ -15,9 +15,23 @@ interface TriggeredPlanSummary {
 }
 
 export interface DeploymentProperties {
-  committer: string | undefined;
-  repositoryAction: string | undefined;
-  repositoryBranchName: string | undefined;
-  repositoryRevisionNumber: string | undefined;
-  repositoryUrl: string | undefined;
+  triggering_event_name: string | undefined;
+  repository_commit_username: string | undefined;
+  repository_action: string | undefined;
+  repository_name: string | undefined;
+  repository_branch_name: string | undefined;
+  repository_url: string | undefined;
+  repository_pull_request_url?: string | undefined;
+  repository_pull_request_number?: number | undefined;
+  repository_pull_request_title?: string | undefined;
+  repository_pull_request_merged_at?: string | undefined;
+  repository_pull_request_created_at?: string | undefined;
```

Comment:
Is there a reason not to use the `PullRequest` type instead?

Reply:
I think the api properties bundle is currently looking for string -> string.

Reply:
I initially had it that way but was getting 500s